modules/ocha_ai_chat/config/install/ocha_ai_chat.settings.yml

```diff
@@ -1,10 +1,20 @@
 defaults:
   form:
+    form_title: 'Ask ReliefWeb'
+    popup_title: 'Ask about this document'
     instructions:
       value: ''
       format: 'plain_text'
     feedback: ''
     formatting: 'basic'
+    answer_min_similarity: 1.25
+    answers:
+      no_document: 'Sorry, no source documents were found.'
+      no_passage: 'Sorry, I could not find information to answer the question.'
+      no_answer: 'Sorry, I was unable to answer your question. Please try again in a short moment.'
+      invalid_answer: 'Sorry, I was unable to answer your question.'
+      document_embedding_error: 'Sorry, there was an error trying to retrieve the documents to the answer to your question.'
```

Comment:
This can be changed in the UI, so doesn't matter much, but could be phrased differently: "Sorry, there was an error trying to retrieve documents to answer your question."
In the next one, there's a typo ("qestion"), and the others could be rewritten to remove "I", and either remove 'in a short moment' or change it to 'shortly'.